packages/client/bin/cli.ts

```diff
@@ -93,8 +93,12 @@ const args = require('yargs')
       number: true,
       default: Config.MAXPEERS_DEFAULT,
     },
-    params: {
-      describe: 'Path to chain parameters json file',
+    customGenesis: {
+      describe: 'Path to custom genesis parameters json file',
```

Comment:
Can you add "from Common" here and below? Otherwise this is lacking context for users and people do not know what type of json file to insert here.

Reply:
Done